### PR TITLE
update draw and modify features example

### PR DESCRIPTION
**Pull request by jipexu**
update draw and modify features example

#13074

### CODE REVIEW

**Comment by jipexu** [comment]
`const extent = get('EPSG:3857').getExtent().slice();
` 
not work ?..

File: examples/draw-and-modify-features.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import {Circle as CircleStyle, Fill, Stroke, Style} from '../src/ol/style.js';
 import {Draw, Modify, Snap} from '../src/ol/interaction.js';
 import {OSM, Vector as VectorSource} from '../src/ol/source.js';
 import {Tile as TileLayer, Vector as VectorLayer} from '../src/ol/layer.js';
+import {get} from 'ol/proj';
```
Comment (ahocevar) [inline examples/draw-and-modify-features.js:7]:
```suggestion
import {get} from 'ol/proj.js';
```

File: examples/draw-and-modify-features.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {Circle as CircleStyle, Fill, Stroke, Style} from '../src/ol/style.js';
 import {Draw, Modify, Snap} from '../src/ol/interaction.js';
 import {OSM, Vector as VectorSource} from '../src/ol/source.js';
 import {Tile as TileLayer, Vector as VectorLayer} from '../src/ol/layer.js';
+import {get} from '../src/ol/proj/projections.js';
```
Comment (ahocevar) [inline examples/draw-and-modify-features.js:7]:
```suggestion
import {get} from '../src/ol/proj.js';
```
The import you had is not part of the API.

File: examples/draw-and-modify-features.js [hunk 3]
```diff
@@ -29,13 +30,17 @@ const vector = new VectorLayer({
...
+extent[2] += extent[2];
 const map = new Map({
   layers: [raster, vector],
   target: 'map',
   view: new View({
     center: [-11000000, 4600000],
     zoom: 4,
-  }),
+    extent,
+  })
```
Comment (ahocevar) [inline examples/draw-and-modify-features.js:43]:
```suggestion
  }),
```

File: examples/draw-and-modify-features.js [hunk 4]
```diff
@@ -29,13 +30,17 @@ const vector = new VectorLayer({
   }),
 });
 
+const extent = get('EPSG:3857').getExtent().slice();
```
Comment (ahocevar) [inline examples/draw-and-modify-features.js:33]:
```suggestion
// Limit multi-world panning to one world east and west of the real world.
// Geometry coordinates have to be within that range.
const extent = get('EPSG:3857').getExtent().slice();
```

**Review by ahocevar — CHANGES_REQUESTED** [review]
Thanks for your work on this, @jipexu. I added a few suggestions which you should be able to commit directly from GitHub.

**Comment by jipexu** [comment]
it's ok or i forget another thing ?

**Review by ahocevar — APPROVED** [review]
Thanks, @jipexu

**Pull request merged**